ptvsd/wrapper.py

```diff
@@ -804,7 +804,6 @@ def close(self):
     def _stop_vsc_message_loop(self):
         self.set_exit()
         self.loop.stop()
-        self.event_loop_thread.join(WAIT_FOR_THREAD_FINISH_TIMEOUT)
```

Comment:
Is there way to perform wait and socket disconnect on a background thread rather than waiting here?